File: Logic Building/Basic Problem/Sum of squares/sum_of_squares.js
function sumOfSquares(n) {
    let sum = 0;
    for(let i=1;i<=n;i++) {
        sum = sum + (i*i);
    }
    return sum;
}
const n = 5;
const result = sumOfSquares(n);
console.log(result);

File: Logic Building/Basic Problem/Sum of squares/sum_of_squares.java
class sum_of_squares {
    public static int sumOfSquares(int n) {
        int sum = 0;

        for(int i=1;i<=n;i++) {
            sum = sum + (i*i);
        }

        return sum;
    }
    public static void main(String[] args) {
        int n = 3;
        int result = sumOfSquares(n);
        System.out.print(result);
    }    
}
